File: app/src/main/java/org/jellyfin/androidtv/ui/home/HomeFragmentRow.java
package org.jellyfin.androidtv.ui.home;

import org.jellyfin.androidtv.ui.presentation.CardPresenter;

import androidx.leanback.widget.ArrayObjectAdapter;

public abstract class HomeFragmentRow {
    abstract public void addToRowsAdapter(CardPresenter cardPresenter, ArrayObjectAdapter rowsAdapter);
}

File: app/src/main/java/org/jellyfin/androidtv/ui/home/HomeFragmentRow.kt
package org.jellyfin.androidtv.ui.home

import android.content.Context
import androidx.leanback.widget.Row
import org.jellyfin.androidtv.ui.presentation.CardPresenter
import org.jellyfin.androidtv.ui.presentation.MutableObjectAdapter

interface HomeFragmentRow {
	fun addToRowsAdapter(context: Context, cardPresenter: CardPresenter, rowsAdapter: MutableObjectAdapter<Row>)
}
